action.yaml

```diff
@@ -1,4 +1,4 @@
-name: 'Trivy Vulnerability Scanner'
+name: 'Aqua Security – Trivy Vulnerability Scanner'
```

Comment:
```suggestion
name: 'Aqua Security Trivy'
```

Reply:
I think that name and description properties are index by GitHub Marketplace search engine so we can keep the name a bit shorter